### PR TITLE
ref #441: align content with label

### PR DESCRIPTION
**Pull request by ckleis**
| Q             | A
| ------------- | ---
| Branch        | features / 6.3.x
| Bug fix?      | yes
| New feature?  | yes/no 
| BC breaks?    | no  
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed issues  | chameleon-system/chameleon-system#441
| License       | MIT

### CODE REVIEW

**Comment by UlrichKu** [comment]
Looks a bit strange/redundant but this is ok as the problem of "aligning the first line of text of arbitrary content" is not really solvable.

Small changes:

I would match the label (col-form-label) to the content on the right: if there is padding (ie for "p-1") this is normally ".25rem".

So: 
`.tableeditcontainer .col-form-label {
    /* matches most content views on the right */

    padding-top: .25rem;
    padding-bottom: .25rem;
}`

And for the contents I would use another class like "form-content-simple" and give that the same padding.

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupDirectory.class.php [hunk 1]
```diff
@@ -116,7 +116,7 @@ public function GetReadOnly()
     {
         $this->GetOptions();
         if (array_key_exists($this->data, $this->options)) {
-            return $this->_GetHiddenField().TGlobal::OutHTML($this->options[$this->data]);
```
Comment (UlrichKu) [inline src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupDirectory.class.php]: The OutHTML should be kept (also below).

File: src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
                 $sTmpFormTabsContent .= '
                 <div class="form-group table-editor-form-group row" id="fieldname_'.TGlobal::OutHTML($oField->name).'">
-                    <div class="col-sm-2 col-form-label">
```
Comment (UlrichKu) [inline src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php]: This padding is the only thing that "col-form-label" does?

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupDirectory.class.php (hunk 1 above)
Comment (ckleis) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupDirectory.class.php]: I don't know how it happened, but of course I added it again.

File: src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php (hunk 2 above)
Comment (ckleis) [reply src/CoreBundle/private/modules/MTTableEditor/views/includes/fields.inc.php]: I used the bootstrap class "col-form-lable" again and overwrote the paddings so it aligns with the standard-fields.

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldOption.class.php [hunk 3]
```diff
@@ -94,7 +94,7 @@ public function GetReadOnly()
     {
         $this->GetOptions();
         if (array_key_exists($this->data, $this->options)) {
-            return $this->_GetHiddenField().TGlobal::OutHTML($this->options[$this->data]);
+            return $this->_GetHiddenField().'<div class="form-content-simple">'.$this->options[$this->data].'</div>';
```
Comment (UlrichKu) [inline src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldOption.class.php]: OutHTML here still missing?

Comment (ckleis) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldOption.class.php]: also added here

**Pull request merged**